### ISSUE TITLE
[1.16.3] Cross-Mod-Conflict - Crash by Startup

### ISSUE THREAD

**Issue by accessdenied0** [JavaPowerFR/StorageTech#4]
Greetings LithimzTV,

I recently tried your mod and I got some issues running it. I'm sure it is some kind of weird mod interaction, but I am not entirely sure what causes it.

I have here the [Crash-Report by startup](https://pastebin.com/25rBp0VM).

[And the log for it, too.](https://pastebin.com/riTLxss4)

I do not have blur installed.

Thank you for your efforts

**Comment by logwet** [comment]
Can confirm that I'm also experiencing the same crash in 1.16.4
<details>
<summary>Crash Report</summary>

```java
---- Minecraft Crash Report ----
// Hi. I'm Minecraft, and I'm a crashaholic.

Time: 15/11/20 2:11 PM
Description: Mod loading error has occurred

java.lang.Exception: Mod Loading has failed
	at net.minecraftforge.fml.CrashReportExtender.dumpModLoadingCrashReport(CrashReportExtender.java:85) ~[forge:?] {re:classloading}
	at net.minecraftforge.fml.client.ClientModLoader.completeModLoading(ClientModLoader.java:188) ~[forge:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.lambda$null$1(Minecraft.java:508) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.util.Util.func_215077_a(Util.java:418) ~[?:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.Minecraft.lambda$new$2(Minecraft.java:504) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.ResourceLoadProgressGui.func_230430_a_(ResourceLoadProgressGui.java:172) ~[?:?] {re:classloading,xf:OptiFine:default}
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:802) ~[?:?] {re:classloading,pl:accesstransformer:B,xf:OptiFine:default}
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:976) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:184) ~[minecraft-1.16.4-client.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_271] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_271] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_271] {}
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_271] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) ~[forge-1.16.4-35.0.17-launcher.jar:35.0] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.0.6.jar:?] {}
	at io.github.zekerzhayard.forgewrapper.installer.Main.main(Main.java:50) [ForgeWrapper-1.4.2.jar:?] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_271] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_271] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_271] {}
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_271] {}
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196) [NewLaunch.jar:?] {}
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231) [NewLaunch.jar:?] {}
	at org.multimc.EntryPoint.listen(EntryPoint.java:143) [NewLaunch.jar:?] {}
	at org.multimc.EntryPoint.main(EntryPoint.java:34) [NewLaunch.jar:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraftforge.registries.NamespacedWrapper.register(NamespacedWrapper.java:56) ~[forge:?] {re:classloading}
-- MOD storagetech --
Details:
	Caused by 0: java.lang.ExceptionInInitializerError
		at javapower.storagetech.core.StorageTech.<init>(StorageTech.java:108) ~[storagetech:1.16.3-R1.0] {re:classloading}
		at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_271] {}
		at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.8.0_271] {}
		at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.8.0_271] {}
		at java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:1.8.0_271] {}
		at java.lang.Class.newInstance(Unknown Source) ~[?:1.8.0_271] {}
		at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:81) ~[forge:35.0] {re:classloading}
		at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:120) ~[forge:?] {re:classloading}
		at java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source) ~[?:1.8.0_271] {}
		at java.util.concurrent.CompletableFuture$AsyncRun.exec(Unknown Source) ~[?:1.8.0_271] {}
		at java.util.concurrent.ForkJoinTask.doExec(Unknown Source) ~[?:1.8.0_271] {}
		at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(Unknown Source) ~[?:1.8.0_271] {}
		at java.util.concurrent.ForkJoinPool.runWorker(Unknown Source) ~[?:1.8.0_271] {re:computing_frames}
		at java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source) ~[?:1.8.0_271] {}

	Mod File: storagetech-1.16.3-R1.1.jar
	Failure message: Storage Tech (storagetech) has failed to load correctly
		java.lang.ExceptionInInitializerError: null
	Mod Version: 1.16.3-R1.1
	Mod Issue URL: NOT PROVIDED
	Exception message: java.lang.IllegalStateException: Can not register to a locked registry. Modder should use Forge Register methods.
Stacktrace:
	at net.minecraftforge.registries.NamespacedWrapper.register(NamespacedWrapper.java:56) ~[forge:?] {re:classloading}
	at net.minecraftforge.registries.NamespacedWrapper.register(NamespacedWrapper.java:72) ~[forge:?] {re:classloading}
	at net.minecraftforge.registries.NamespacedWrapper.func_218381_a(NamespacedWrapper.java:40) ~[forge:?] {re:classloading}
	at net.minecraft.util.registry.Registry.func_218322_a(Registry.java:456) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.util.registry.Registry.func_218325_a(Registry.java:452) ~[?:?] {re:classloading,pl:accesstransformer:B}
	at net.minecraft.item.crafting.IRecipeSerializer.func_222156_a(IRecipeSerializer.java:39) ~[?:?] {re:classloading}
	at javapower.storagetech.setup.CommonSetup.<clinit>(CommonSetup.java:34) ~[storagetech:1.16.3-R1.0] {re:classloading}
	at javapower.storagetech.core.StorageTech.<init>(StorageTech.java:108) ~[storagetech:1.16.3-R1.0] {re:classloading}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_271] {}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.8.0_271] {}
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.8.0_271] {}
	at java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:1.8.0_271] {}
	at java.lang.Class.newInstance(Unknown Source) ~[?:1.8.0_271] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:81) ~[forge:35.0] {re:classloading}
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:120) ~[forge:?] {re:classloading}
	at java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source) ~[?:1.8.0_271] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(Unknown Source) ~[?:1.8.0_271] {}
	at java.util.concurrent.ForkJoinTask.doExec(Unknown Source) ~[?:1.8.0_271] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(Unknown Source) ~[?:1.8.0_271] {}
	at java.util.concurrent.ForkJoinPool.runWorker(Unknown Source) ~[?:1.8.0_271] {re:computing_frames}
	at java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source) ~[?:1.8.0_271] {}


-- System Details --
Details:
	Minecraft Version: 1.16.4
	Minecraft Version ID: 1.16.4
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_271, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 1473160104 bytes (1404 MB) / 4294967296 bytes (4096 MB) up to 4294967296 bytes (4096 MB)
	CPUs: 4
	JVM Flags: 9 total; -XX:+UseG1GC -XX:+UnlockExperimentalVMOptions -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms4096m -Xmx4096m
	ModLauncher: 8.0.6+85+master.325de55
	ModLauncher launch target: fmlclient
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.2.jar mixin PLUGINSERVICE 
		/eventbus-3.0.5-service.jar eventbus PLUGINSERVICE 
		/forge-1.16.4-35.0.17-launcher.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.16.4-35.0.17-launcher.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-2.2.0-shadowed.jar accesstransformer PLUGINSERVICE 
		/forge-1.16.4-35.0.17-launcher.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.16.4-35.0.17-launcher.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.2.jar mixin TRANSFORMATIONSERVICE 
		/OptiFine_1.16.4_HD_U_G5.jar OptiFine TRANSFORMATIONSERVICE 
		/OptiForge-MC1.16.4-0.4.2.jar optiforge TRANSFORMATIONSERVICE 
		/forge-1.16.4-35.0.17-launcher.jar fml TRANSFORMATIONSERVICE 
		/OptiForge-MC1.16.4-0.4.2.jar FakeOptiFine TRANSFORMATIONSERVICE 
		/OptiForge-MC1.16.4-0.4.2.jar optiforgewrapper TRANSFORMATIONSERVICE 
	FML: 35.0
	Forge: net.minecraftforge:35.0.17
	FML Language Providers: 
		javafml@35.0
		minecraft@1
	Mod List: 
		rsrequestify-1.16.3-2.0.1.jar                     |RSRequestify                  |rsrequestify                  |2.0.1               |CREATE_REG|NOSIGNATURE
		QuarkOddities-1.16.3.jar                          |Quark Oddities                |quarkoddities                 |1.16.3              |CREATE_REG|NOSIGNATURE
		ClientTweaks_1.16.3-5.2.0.jar                     |Client Tweaks                 |clienttweaks                  |5.2.0               |CREATE_REG|NOSIGNATURE
		NetherPortalFix_1.16.3-7.2.1.jar                  |NetherPortalFix               |netherportalfix               |7.2.1               |CREATE_REG|NOSIGNATURE
		jei-1.16.4-7.6.0.53.jar                           |Just Enough Items             |jei                           |7.6.0.53            |CREATE_REG|NOSIGNATURE
		OptiForge-MC1.16.4-0.4.2.jar                      |OptiForge                     |optiforge                     |0.4.2               |CREATE_REG|NOSIGNATURE
		AttributeFix-1.16.1-8.0.1.jar                     |AttributeFix                  |attributefix                  |8.0.1               |CREATE_REG|NOSIGNATURE
		Mekanism-1.16.4-10.0.17.444.jar                   |Mekanism                      |mekanism                      |10.0.17             |CREATE_REG|NOSIGNATURE
		invtweaks-1.16.2-1.0.0.jar                        |Inventory Tweaks Renewed      |invtweaks                     |1.16.2-1.0.0        |CREATE_REG|NOSIGNATURE
		Neat+1.7-24.jar                                   |Neat                          |neat                          |1.7-24              |CREATE_REG|NOSIGNATURE
		ForgeEndertech-1.16.3-7.0.6.0-build.0024.jar      |Forge Endertech               |forgeendertech                |7.0.6.0             |CREATE_REG|NOSIGNATURE
		CTM-MC1.16.1-1.1.1.5.jar                          |ConnectedTexturesMod          |ctm                           |MC1.16.1-1.1.1.5    |CREATE_REG|NOSIGNATURE
		Controlling-7.0.0.11.jar                          |Controlling                   |controlling                   |7.0.0.11            |CREATE_REG|NOSIGNATURE
		ReAuth-1.16-Forge-3.9.3.jar                       |ReAuth                        |reauth                        |3.9.3               |CREATE_REG|3d:06:1e:e5:da:e2:ff:ae:04:00:be:45:5b:ff:fd:70:65:00:67:0b:33:87:a6:5f:af:20:3c:b6:a1:35:ca:7e
		Placebo-1.16.3-4.3.3.jar                          |Placebo                       |placebo                       |4.3.3               |CREATE_REG|NOSIGNATURE
		mightyarchitect-mc1.16.3_v0.5.jar                 |The Mighty Architect          |mightyarchitect               |0.5                 |CREATE_REG|NOSIGNATURE
		ftb-gui-library-1603.1.1.25.jar                   |FTB GUI Library               |ftbguilibrary                 |1603.1.1.25         |CREATE_REG|NOSIGNATURE
		mdecore-1.16.3-16.0.2.jar                         |MattDahEpic Core              |mdecore                       |16.0.2              |CREATE_REG|NOSIGNATURE
		Bookshelf-1.16.4-9.0.7.jar                        |Bookshelf                     |bookshelf                     |9.0.7               |CREATE_REG|ea:45:b3:82:b6:9d:50:16:95:e7:2e:34:e1:92:d5:b4:9b:69:90:d3:4f:2e:71:99:b0:be:40:80:27:1f:3e:b0
		buildinggadgets-3.7.2.jar                         |Building Gadgets              |buildinggadgets               |3.7.2               |CREATE_REG|NOSIGNATURE
		storagetech-1.16.3-R1.1.jar                       |Storage Tech                  |storagetech                   |1.16.3-R1.1         |ERROR     |NOSIGNATURE
		DataFixerSlayer+1.0-2.jar                         |DataFixerSlayer               |datafixerslayer               |1.0-2               |CREATE_REG|NOSIGNATURE
		MekanismGenerators-1.16.4-10.0.17.444.jar         |Mekanism: Generators          |mekanismgenerators            |10.0.17             |CREATE_REG|NOSIGNATURE
		Toast-Control-1.16.3-4.3.0.jar                    |Toast Control                 |toastcontrol                  |4.3.0               |CREATE_REG|NOSIGNATURE
		JEITweaker-1.16.3-1.0.1.4.jar                     |JEI Tweaker                   |jeitweaker                    |1.0.1.4             |CREATE_REG|NOSIGNATURE
		FpsReducer-mc1.16.3-1.18.jar                      |FPS Reducer                   |fpsreducer                    |mc1.16.3-1.18       |CREATE_REG|NOSIGNATURE
		JustEnoughResources-1.16.3-0.12.0.100.jar         |Just Enough Resources         |jeresources                   |0.12.0.100          |CREATE_REG|NOSIGNATURE
		CraftTweaker-1.16.4-7.0.0.59.jar                  |CraftTweaker                  |crafttweaker                  |7.0.0.59            |CREATE_REG|NOSIGNATURE
		BetterThanMending-1.5.1.jar                       |BetterThanMending             |betterthanmending             |1.5.1               |CREATE_REG|NOSIGNATURE
		ftb-chunks-1603.2.0.43.jar                        |FTB Chunks                    |ftbchunks                     |1603.2.0.43         |CREATE_REG|NOSIGNATURE
		forge-1.16.4-35.0.17-universal.jar                |Forge                         |forge                         |35.0.17             |CREATE_REG|22:af:21:d8:19:82:7f:93:94:fe:2b:ac:b7:e4:41:57:68:39:87:b1:a7:5c:c6:44:f9:25:74:21:14:f5:0d:90
		refinedstorage-1.9.9.jar                          |Refined Storage               |refinedstorage                |1.9.9               |CREATE_REG|NOSIGNATURE
		ironchest-1.16.4-11.2.10.jar                      |Iron Chests                   |ironchest                     |1.16.4-11.2.10      |CREATE_REG|NOSIGNATURE
		CraftingTweaks_1.16.3-12.2.0.jar                  |Crafting Tweaks               |craftingtweaks                |12.2.0              |CREATE_REG|NOSIGNATURE
		forge-1.16.4-35.0.17-client.jar                   |Minecraft                     |minecraft                     |1.16.4              |CREATE_REG|NOSIGNATURE
		cofh_core-1.16.3-1.0.4.jar                        |CoFH Core                     |cofh_core                     |1.0.4               |CREATE_REG|NOSIGNATURE
		thermal-1.16.3-1.0.5.jar                          |Thermal Series                |thermal                       |1.0.5               |CREATE_REG|NOSIGNATURE
		industrial-foregoing-1.16.3-3.1.1-a834e76.jar     |Industrial Foregoing          |industrialforegoing           |3.1.1               |CREATE_REG|NOSIGNATURE
		theoneprobe-1.16-3.0.6.jar                        |The One Probe                 |theoneprobe                   |1.16-3.0.6          |CREATE_REG|NOSIGNATURE
		AmbientSounds_v3.0.21_mc1.16.2.jar                |Ambient Sounds                |ambientsounds                 |3.0.3               |CREATE_REG|NOSIGNATURE
		MouseTweaks-2.13-mc1.16.2.jar                     |Mouse Tweaks                  |mousetweaks                   |2.13                |CREATE_REG|NOSIGNATURE
		titanium-1.16.4-3.2.0.jar                         |Titanium                      |titanium                      |3.2.0               |CREATE_REG|NOSIGNATURE
		Psi+1.16-88.jar                                   |Psi                           |psi                           |1.16-88             |CREATE_REG|NOSIGNATURE
		MekanismAdditions-1.16.4-10.0.17.444.jar          |Mekanism: Additions           |mekanismadditions             |10.0.17             |CREATE_REG|NOSIGNATURE
		Lollipop-1.16.3-3.2.2.jar                         |Lollipop                      |lollipop                      |3.2.2               |CREATE_REG|NOSIGNATURE
		AdLods-1.16.3-4.1.1.0-build.0026.jar              |Large Ore Deposits            |adlods                        |4.1.1.0             |CREATE_REG|NOSIGNATURE
		shulker_enchantments-1.1.1.jar                    |Shulker Enchantments          |shulker_enchantments          |1.1.1               |CREATE_REG|NOSIGNATURE
		jeiintegration_1.16.3-6.0.0.8.jar                 |JEI Integration               |jeiintegration                |6.0.0.8             |CREATE_REG|NOSIGNATURE
		Patchouli-1.16.2-47.jar                           |Patchouli                     |patchouli                     |1.16.2-47           |CREATE_REG|NOSIGNATURE
		ftb-backups-2.1.1.6.jar                           |FTB Backups                   |ftbbackups                    |2.1.1.6             |CREATE_REG|NOSIGNATURE
		serverconfigupdater-1.3.jar                       |ServerConfig Updater          |serverconfigupdater           |1.3                 |CREATE_REG|NOSIGNATURE
		FastWorkbench-1.16.3-4.4.0.jar                    |FastWorkbench                 |fastbench                     |4.4.0               |CREATE_REG|NOSIGNATURE
		polymorph-forge-1.16.3-0.15.jar                   |Polymorph                     |polymorph                     |1.16.3-0.15         |CREATE_REG|NOSIGNATURE
		AutoRegLib-1.6-46.jar                             |AutoRegLib                    |autoreglib                    |1.6-46              |CREATE_REG|NOSIGNATURE
		Quark-r2.4-274.jar                                |Quark                         |quark                         |r2.4-274            |CREATE_REG|NOSIGNATURE
		StorageDrawers-1.16.3-8.2.1.jar                   |Storage Drawers               |storagedrawers                |8.2.1               |CREATE_REG|NOSIGNATURE
		thermal_expansion-1.16.3-1.0.5.jar                |Thermal Expansion             |thermal_expansion             |1.0.5               |CREATE_REG|NOSIGNATURE
		elevatorid-1.16.4-1.7.8.jar                       |Elevator Mod                  |elevatorid                    |1.16.4-1.7.8        |CREATE_REG|NOSIGNATURE
		InventoryHud_1.16.x.forge-3.1.4.jar               |Inventory HUD+(Forge edition) |inventoryhud                  |3.1.4               |CREATE_REG|NOSIGNATURE
		ExtraStorage-1.16.4-1.2.1.jar                     |Extra Storage                 |extrastorage                  |1.2.1               |CREATE_REG|NOSIGNATURE
		thermal_locomotion-1.16.3-1.0.3.jar               |Thermal Locomotion            |thermal_locomotion            |1.0.3               |CREATE_REG|NOSIGNATURE
		MekanismTools-1.16.4-10.0.17.444.jar              |Mekanism: Tools               |mekanismtools                 |10.0.17             |CREATE_REG|NOSIGNATURE
		FastFurnace-1.16.3-4.3.0.jar                      |FastFurnace                   |fastfurnace                   |4.3.0               |CREATE_REG|NOSIGNATURE
		AppleSkin-mc1.16.2-forge-1.0.14.jar               |AppleSkin                     |appleskin                     |1.0.14              |CREATE_REG|NOSIGNATURE
		cc-tweaked-1.16.4-1.94.0.jar                      |CC: Tweaked                   |computercraft                 |1.94.0              |CREATE_REG|NOSIGNATURE
		AI-Improvements-1.16.2-0.3.0.jar                  |AI-Improvements               |aiimprovements                |0.3.0               |CREATE_REG|NOSIGNATURE
		moreoverlays-1.18.10-mc1.16.3.jar                 |More Overlays Updated         |moreoverlays                  |1.18.10-mc1.16.3    |CREATE_REG|NOSIGNATURE
		globalxp-1.16.3-v1.4.15.jar                       |Global XP                     |globalxp                      |v1.4.15             |CREATE_REG|NOSIGNATURE
		refinedstorageaddons-0.7.2.jar                    |Refined Storage Addons        |refinedstorageaddons          |0.7.2               |CREATE_REG|NOSIGNATURE
		MineMenu-1.16.4-1.8.7.jar                         |MineMenu                      |minemenu                      |1.16.4-1.8.7        |CREATE_REG|NOSIGNATURE
		FastLeafDecay-v25.jar                             |FastLeafDecay                 |fastleafdecay                 |v25                 |CREATE_REG|NOSIGNATURE
		BetterAdvancements-1.16.4-0.1.0.103.jar           |Better Advancements           |betteradvancements            |0.1.0.103           |CREATE_REG|NOSIGNATURE
	Crash Report UUID: 99f55045-857b-4b5a-b74b-d80a0dc4cf1e
	OptiFine Version: OptiFine_1.16.4_HD_U_G5
	OptiFine Build: 20201106-171901
	Render Distance Chunks: 12
	Mipmaps: 4
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: null
	OpenGlVersion: 4.6.0 NVIDIA 456.71
	OpenGlRenderer: GeForce GTX 1050/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 4
```
</details>
Thanks for your work on this. I'm interested in trying out the Mekanism chemicals integration 🙂

**Comment by hodo1962** [comment]
I have the same error. 
Crash report is the same as above.

[crash-2020-11-22_20.20.55-fml.txt](https://github.com/JavaPowerFR/StorageTech/files/5580145/crash-2020-11-22_20.20.55-fml.txt)

**Comment by JavaPowerFR** [comment]
Hi, We have fixed the issue in the version [1.16.3-R1.2](https://www.curseforge.com/minecraft/mc-mods/storage-tech/files/3120502) the recipe serializer that be wrong registered.

thanks you !
